File: Programmation/11_09_13/max-premier-prog.rb
#!/usr/bin/env ruby

puts "Entrez un nom commun masculin singulier : "
$nom = gets.chomp


$pluriel = $nom + "s" # well, let's say that most of the time it works

puts "Les flexions de #{$nom} sont :
un #{$nom},
des #{$pluriel}\n" 

File: Programmation/11_09_13/max-premier-prog.pl
#!/usr/bin/perl -w

use strict ;
use locale ;

# variables you have to declare
# because of 'use strict'
my ($nom, $pluriel) ;

print "Entrez un nom commun masculin singulier : " ;
chomp($nom = <STDIN>);

$pluriel = $nom . "s" ; # well, let's say that most of the time it works

print "Les flexions de $nom sont :
un $nom,
des $pluriel \n" ;
